app/views/galleries/show.rb

```diff
@@ -11,6 +11,12 @@ def page_title
       mustache[:page_title] ||= [@gallery.title, site_title].join(' - ')
     end
 
+    def head_links
+      mustache[:head_links] ||= begin
+        { items: [{ rel: 'alternate', type: 'application/rss+xml', href: galleries_url(format: 'rss') }] + super[:items] }
```

Comment:
This would lose anything else coming from `super` not in the element keyed `:items`